yt/fields/magnetic_field.py

```diff
@@ -119,59 +120,121 @@ def _magnetic_field_toroidal(field, data):
             phi = data["index", "spherical_phi"]
             return get_sph_phi_component(Bfields, phi, normal)
 
+        def _magnetic_field_poloidal_magnitude(field, data):
```

Comment:
Why do we need to put "magnitude" on this field component? We don't do that for any other vector components. Is there a reason we can't give this component a direction? (I see that the way it's computed here doesn't allow you to).

Reply:
Well, because it's not a component, it's really the amplitude of the poloidal projection of the total field. That's what myself, and my team, as users, expect to get when querying `B_poloidal`.

Reply:
Ok sounds good